src/defaults.js

```diff
@@ -834,7 +832,7 @@ var defaults = {
  * @see https://developer.mozilla.org/en-US/docs/Web/CSS/font
  */
 
-var textWidthCache = new LRUCache(2000);
+var fontMetrics = {};
 
```

Comment:
The LRU cache cached the most recent 2000 strings (cell values) and their pixel widths. We're not doing that any more. The new measure is so much faster it's probably not worth it. Only really helped on a static screen, since scrolling would wipe those 2000 in short order. But on a static screen, I believe our partial render design will obviate the need for this. See what I mean?